pyrpl/widgets/module_widgets/iir_widget.py

```diff
@@ -9,9 +9,9 @@
 import sys
 from ... import APP
 
-class MyGraphicsWindow(pg.GraphicsWindow):
+class MyGraphicsLayoutWidget(pg.GraphicsLayoutWidget):
     def __init__(self, title, parent):
-        super(MyGraphicsWindow, self).__init__(title)
+        super(MyGraphicsLayoutWidget, self).__init__(parent, title=title)
         self.parent = parent
```

Comment:
parent has already been set in the line above.  I make this more explicit by:
super().__init__(title=title, parent=parent)